modules/oc-by-example-content.adoc

```diff
@@ -1857,10 +1857,39 @@ Log in to a server
   oc login localhost:8443 --web --callback-port 8280
 
   # Log in to the external OIDC issuer through Auth Code + PKCE by starting a local server listening port 8080
-  oc login localhost:8443 --exec-plugin=oc-oidc --client-id=client-id --extra-scopes=email,profile --callback-port=8080
+  oc login --exec-plugin=oc-oidc --issuer-url=<issuer_url> --client-id=<client_id> --extra-scopes=email,profile --callback-port=8080
+
+ifdef::openshift-dedicated,openshift-rosa[]
```

Comment:
@jneczypor I'm not sure whether `openshift-dedicated,openshift-rosa` condition is discussed / agreed on by you and others. I assume yes. Given so, line 1860 needs to be moved down within the `ifdef` too, because line 1860 is also about external oidc feature (i.e. `without oauth server` as PR title).